int.env

```diff
@@ -0,0 +1,29 @@
+APACHE_BASE_PATH=main
+ALTI_URL=//service-alti.int.bgdi.ch
+API_URL=//mf-chsdi3.int.bgdi.ch
+SHOP_URL=//shop-bgdi.int.bgdi.ch
+DBHOST=pg-sandbox.bgdi.ch
+DBPORT=5432
+DBSTAGING=int
+DEPLOY_TARGET=int
+GEODATA_STAGING=integration
+GEOADMIN_FILE_STORAGE_BUCKET=public-int-bgdi-ch
+GEOADMINHOST=mf-geoadmin3.int.bgdi.ch
+HOST=mf-chsdi3.int.bgdi.ch
+KEEP_VERSION=true
+WMTS_PUBLIC_HOST=tod.int.bgdi.ch
+MODWSGI_CONFIG=production.ini
```

Comment:
Is there a integration.ini instead?

Reply:
No, on `prod` and `int`  the `production.ini` is used. On `dev` it's `development.ini` (which sources `production.ini`). Again, don't ask why